### PR TITLE
Topic fixes

### PR DESCRIPTION
**Pull request by diegorosolino**
### Ticket
[Change the transaction link icon and add a share comment link](https://www.notion.so/usedispatch/Change-the-transaction-link-icon-and-add-a-share-comment-link-c736248b7138421195942b5dd2f5ac8e)

### Description
This pr changes the transaction link icon and adds to share a comment

### Screenshots
Share:
localhost_3000_forum_83V8YXXd7AURNWRAtqeoGsnAbQk16zr68TAWZw5bSzfF_topic_1_cluster=devnet(iPhone SE) (1)
localhost_3000_forum_83V8YXXd7AURNWRAtqeoGsnAbQk16zr68TAWZw5bSzfF_topic_1_cluster=devnet(iPhone SE)

https://user-images.githubusercontent.com/57630050/196980025-bc3cd588-2a12-446e-a1f4-ff72b57828d1.mp4

Transaction link:

https://user-images.githubusercontent.com/57630050/196979850-1bb07856-0b7f-4082-be45-d98774c14d14.mp4

### CODE REVIEW

**Review by ana2201 — CHANGES_REQUESTED** [review]
If the user clicks on more than one link icon, and then pastes the link, the result is an url with the # concatenated. It should have only the last one, so that when the url is pasted, the page scrolls into the selected post

File: src/components/forums/topic/SharePost.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { Chain } from '../../../assets';
+import { MessageType } from './../../common';
+import { Notification } from '../../../components/forums';
+import { NOTIFICATION_BANNER_TIMEOUT } from '../../../utils/consts';
+
+interface SharePostProps {
+  postAddress: string;
+}
+
+const SharePost = ({ postAddress }: SharePostProps): JSX.Element => {
```
Comment (ana2201) [inline src/components/forums/topic/SharePost.tsx:12]: Maybe define it as `export function SharePost` just so we define all the components the same way

**Comment by ana2201** [comment]
By the way, it's great that you created new components for the repeated logic 🙂

File: src/components/forums/topic/SharePost.tsx (hunk 1 above)
Comment (diegorosolino) [reply src/components/forums/topic/SharePost.tsx]: Okey, I changed all components that I defined for this form

File: src/assets/ExternalLink.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export function ExternalLink()  {
```
Comment (ana2201) [inline src/assets/ExternalLink.tsx:1]: add return type to the function (`: JSX.Element`)

**Review by ana2201 — APPROVED** [review]
Minor comment, but otherwise looks good to me 🙂

**Pull request merged**